packages/devops/configs/servers.conf

```diff
@@ -74,6 +74,50 @@ server {
         internal;
   }
 
+}
+server {
+
+  listen       443 ssl http2;
+  listen       [::]:443 ssl http2;
+  server_name  ~^.*psss-api\.tupaia\.org$;
```

Comment:
not actually related to your changes at all, but this reminds me that we need to add `psss` to the reserved endings in `validateBranchName.sh`

Reply:
Oh true - I might as well slip this into the next PR!